metricbeat/module/nats/connections/_meta/data.json

```diff
@@ -0,0 +1,21 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"nats",
+        "name":"connections",
+        "rtt":44269
+    },
+    "nats":{
+        "server_id": "bUAdpRFtMWddIBWw80Yd9D",
```

Comment:
Do you expect more info about the server then just the id? If yes, perhaps we should make this `server.id`.

Reply:
I had similar thought about merging `server_id` and `now` fileds in something like:
```
{
 "server": {
    "id": "foobar",
    "reporting_time": ....
 }
}
```
cc: @skatsaounis, @MichaelKatsoulis

Reply:
I like the idea. Is it reporting_time or is this actually the time of the server? `server.time`?

Reply:
Basically, it's the same thing. So let's keep it as `sever.time` for simplicity.

Reply:
@ChrsMark Can you follow up on this and also do the change for the changelog?